src/components/OrderConfirmation/OrderConfirmation.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { OrderItem } from '..';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+
+export class OrderConfirmation extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = { orderOwner: '' };
```

Comment:
Don't think this ever gets used as its setting state on its parent

Reply:
I'm sure I got errors when I tried to remove this, I shall reattempt though - cause as you say I thought it shouldn't be needed 👍